.pipelines/build.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
   - script: |
       sudo apt-get install gcc-mingw-w64
       mkdir manifest
+      make test-security
```

Comment:
we can add the test-security target to all. This way we don't have to add a separate command